terraform/20-app/ecs.service.feature-flags.tf

```diff
@@ -89,6 +89,13 @@ module "ecs_service_feature_flags" {
     }
   }
 
+  task_exec_iam_statements = {
+    kms_keys = {
+      actions   = ["kms:Decrypt"]
+      resources = [module.kms_secrets_app_operator.key_arn]
```

Comment:
These are set to be locked down to only the groups of secrets that each service needs. E.g. the feature flags service only needs the secrets used by operators i.e. the feature flag api keys and login secrets